.github/workflows/fmt.yml

```diff
@@ -1,11 +1,24 @@
-on: [push, pull_request, pull_request_target]
+on: push
```

Comment:
Given the nature of this workflow, I think it's okay to run it on `push` only -- this should also cover pushes made in fork PRs

Reply:
Do you want to exclude the default branch if we're running on every `push` event?

Reply:
Good call! I'll add that